graphql/src/query.rs

```diff
@@ -288,6 +289,18 @@ impl Mutation {
             .await?;
         Ok(true)
     }
+
+    async fn update_source_text(
```

Comment:
Hm I think the name `update_word` is better since we'll support more fields in the future. For new things I want to use the term `word` instead of `form` because it's just more obvious to people.

Reply:
That makes sense, I've went ahead and made the change!